test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetLocalsCommandTest.cs

```diff
@@ -33,6 +32,21 @@ public void LocalsCommand_Success_InvalidArguments_HelpMessage(string args)
             Util.VerifyResultSuccess(result, LocalsHelpStringFragment);
         }
 
+        [Theory]        
+        [InlineData("locals all -list extraArg")]
```

Comment:
A `Theory` is supposed to be used to run multiple use cases for the same test. Either add more test cases or change this test to be a `Fact`